test/sql/index_scan_sql_test.cpp

```diff
@@ -99,137 +99,6 @@ TEST_F(IndexScanSQLTests, CreateIndexAfterInsertOnMultipleColumnsTest) {
   catalog::Catalog::GetInstance()->DropDatabaseWithName(DEFAULT_DB_NAME, txn);
   txn_manager.CommitTransaction(txn);
 }
-TEST_F(IndexScanSQLTests, SQLTest) {
```

Comment:
Does these test corrupt?

Reply:
Actually we are moving to build junit test for SQL. Thus most tests here will be abandoned in the final pull request.